core/src/main/java/org/stellar/anchor/sep10/Sep10Service.java

```diff
@@ -224,14 +224,17 @@ public String validateChallenge(String challengeXdr)
       signers.add(challenge.getClientAccountId());
 
       infoF(
-          "Verifying challenge threshold. server_account={}, signers={}",
+          "Verifying challenge threshold. server_account={}, client_domain={}, signers={}",
           shorter(serverAccountId),
+          clientDomain,
           signers.size());
 
       if ((clientDomain != null && challenge.getTransaction().getSignatures().size() != 3)
           || (clientDomain == null && challenge.getTransaction().getSignatures().size() != 2)) {
         infoF(
-            "Invalid SEP 10 challenge exception, there is more than one client signer on challenge transaction for an account that doesn't exist");
+            "ALERT: Invalid SEP 10 challenge exception, there is more than one client signer on challenge transaction for an account that doesn't exist. client_domain={}, account_id={}",
```

Comment:
`ALERT:` is MGI specific and lack of consistency. We should avoid anchor-specific implementation.

Reply:
Yep sorry was just a copy-paste